templates/internet-of-things/smart-city.html

```diff
@@ -126,533 +136,518 @@ <h1>Build the future with smart cities</h1>
+                in
+              government services&nbsp;&rsaquo;</a>
+            </div>
           </div>
         </div>
+      </div>
+    </div>
+  </section>
+
+  <section class="p-section">
+    <div class="row--50-50">
+      <hr class="p-rule" />
+      <div class="col">
+        <h2>Let's work together</h2>
+      </div>
+      <div class="col">
         <p>
-          <a href="/engage/open-source-security-cloud-edge-webinar">Securing open source from cloud to edge&nbsp;&rsaquo;</a>
+          We work closely with our customers to solve complex problems together. Our talented engineers are
+          experts in IoT and are well-qualified to provide expertise and guidance to both smaller businesses and larger enterprises.
         </p>
+        <div class="p-cta-block">
+          <a href="#get-in-touch" class="p-button--positive js-invoke-modal">Get in touch</a>
+        </div>
       </div>
     </div>
+  </section>
 
-    <div class="u-fixed-width">
+  <section class="p-section--deep">
+    <div class="row p-section--shallow">
       <hr class="p-rule" />
+      <h2>Explore the use cases</h2>
     </div>
-
-    <div class="row">
-      <div class="col-4">
-        <div class="p-heading-icon">
-          <div class="p-heading-icon__header">
-            {{ image(url="https://assets.ubuntu.com/v1/b061c401-White+paper.svg",
+    <div class="p-equal-height-row--wrap has-divider-3">
+      <div class="p-equal-height-row__col">
+        <div class="p-equal-height-row__item">
+          <div class="p-image-container--3-2-on-small p-image-container--square-on-medium p-image-container--2-3-on-large is-highlighted">
+            {{ image(url="https://assets.ubuntu.com/v1/24457f3e-signage.png",
                         alt="",
-                        width="171",
-                        height="150",
+                        width="852",
+                        height="1279",
                         hi_def=True,
-                        loading="lazy",
-                        attrs={"class": "p-heading-icon__img"},) | safe
+                        loading="auto|lazy",
+                        attrs={"class": "p-image-container__image"}) | safe
             }}
-            <h4 class="p-heading-icon__title p-heading--5">Whitepaper</h4>
           </div>
         </div>
-        <p>
-          <a href="/engage/gov-webinar">Open source in Government&nbsp;&rsaquo;</a>
-        </p>
+        <div class="p-equal-height-row__item">
+          <hr class="p-rule--highlight u-hide--medium u-hide--small" />
+          <h3 class="p-heading--5">Signage</h3>
+        </div>
+        <p class="p-equal-height-row__item">
+          A small footprint and full OpenGL make Ubuntu Core the perfect platform for secure digital signs.
+        </p>
+        <div class="p-equal-height-row__item">
+          <hr class="p-rule--muted u-hide--large" />
+          <p>
+            <a href="/internet-of-things/smart-displays">Learn more&nbsp;&rsaquo;</a>
```

Comment:
please add `aria-label` to each one of these "Learn more" links, otherwise it won't be clear to people with limited sight what these links are for